modules/base/src/components/options/Option.vue

```diff
@@ -0,0 +1,350 @@
+        :record="record"
+        :successState="successState"
+        :errorState="errorState"
+      ></OptionStringArrayCountries>
+      <OptionStringArray
+        v-else-if="record.OptType == optTypeStringArray"
+        :record="record"
+        :successState="successState"
+        :errorState="errorState"
+      ></OptionStringArray>
+      <OptionStringSelect
+        v-else-if="record.ExternalOptType == 'string list'"
+        :record="record"
+        :successState="successState"
+        :errorState="errorState"
+      ></OptionStringSelect>
+      <OptionString v-else :record="record" :successState="successState" :errorState="errorState"></OptionString>
+
+      <div v-if="validationError" class="ui error message">
+        <div class="header">Validation failed</div>
+        <p>{{ validationError }}</p>
+      </div>
+      <div v-else-if="request.error" class="ui error message">
+        <div class="header">Rejected by Portmaster</div>
+        <p>{{ request.error }}</p>
+      </div>
+    </div>
+
+    <!-- Default Value -->
+    <div v-bind:class="[!activeState() ? 'active' : 'inactive', 'column']">
+      <i v-bind:class="[!activeState() ? 'green' : 'grey outline', 'circle icon']"></i>
+
+      <OptionBoolean v-if="record.OptType == optTypeBool" :record="record" :displayDefault="true"></OptionBoolean>
+      <OptionIntSecurityLevel
+        v-else-if="record.ExternalOptType == 'security level'"
+        :record="record"
+        :displayDefault="true"
+      ></OptionIntSecurityLevel>
+      <OptionInt v-else-if="record.OptType == optTypeInt" :record="record" :displayDefault="true"></OptionInt>
+      <OptionStringArray
+        v-else-if="record.OptType == optTypeStringArray"
+        :record="record"
+        :displayDefault="true"
+      ></OptionStringArray>
+      <OptionString v-else :record="record" :displayDefault="true"></OptionString>
+    </div>
+
+    <div v-if="showHelp" class="sixteen wide column">
+      <div v-if="record.Help" class="ui raised segment help-popup">
+        <div class="ui top left attached label">Input Help for {{ record.Name }}</div>
+        <div v-on:click="showHelp = false" class="ui top right attached label" style="cursor: pointer;">
+          <i class="times icon"></i>
+        </div>
+        <p>
+          {{ record.Help }}
+        </p>
+      </div>
+
+      <!-- TODO: replace with interactive version -->
+      <div v-else-if="rKey == 'config:filter/lists'" class="ui raised segment help-popup">
+        <div class="ui top left attached label">Input Help for {{ record.Name }}</div>
+        <div v-on:click="showHelp = false" class="ui top right attached label" style="cursor: pointer;">
+          <i class="times icon"></i>
+        </div>
+        <p>
+          Categories:<br /><br />
+          <strong>TRAC</strong>: Ads & Trackers<br />
+          Services that track and profile people online.<br />
+          <span style="margin-left: 20px;"></span><strong>ADS</strong>: Ads<br />
```

Comment:
Although is is a temporary solution, prefer CSS classes over `style=""` tags because the will cause confusion when debugging styles. In addition, \<strong> is for styling here and should be moved to CSS.

I.e
```html
<p calss="help">
    <span>ADS</span><span>: Ads</span>
<!-- ... -->
</p>
```

```css
p.help {
    span:first-of-type {
        margin-left: 20px;
        font-weight: bold;
    }
```

Reply:
Ah, very nice way of putting things. I won't improve, as this will be removed soon anyway.